rosconsole: compile failure

Is anyone else seeing this failure on the latest meta-ros? I have suspicion that there are tonnes of other packages somehow mysteriously began failing since a couple days ago on my build machines. 

```
In file included from /media/build/yocto/build/tmp/sysroots/qemuarm/usr/include/log4cxx/level.h:22:0,
                 from /media/build/yocto/build/tmp/work/armv5te-poky-linux-gnueabi/rosconsole/1.10.2-r0/ros_comm-1.10.2/tools/rosconsole/include/ros/console.h:45,
                 from /media/build/yocto/build/tmp/work/armv5te-poky-linux-gnueabi/rosconsole/1.10.2-r0/ros_comm-1.10.2/tools/rosconsole/src/rosconsole/impl/rosconsole_log4cxx.cpp:36:
/media/build/yocto/build/tmp/sysroots/qemuarm/usr/include/log4cxx/logstring.h:28:29: fatal error: log4cxx/log4cxx.h: No such file or directory
 #include <log4cxx/log4cxx.h>
                             ^
compilation terminated.
make[2]: *** [CMakeFiles/rosconsole_log4cxx.dir/src/rosconsole/impl/rosconsole_log4cxx.cpp.o] Error 1
make[2]: Leaving directory `/media/build/yocto/build/tmp/work/armv5te-poky-linux-gnueabi/rosconsole/1.10.2-r0/build'
make[1]: *** [CMakeFiles/rosconsole_log4cxx.dir/all] Error 2
make[1]: Leaving directory `/media/build/yocto/build/tmp/work/armv5te-poky-linux-gnueabi/rosconsole/1.10.2-r0/build'
make: *** [all] Error 2
```

[Full Log](http://paste.ubuntu.com/7308529/)

--- a/classes/autotools-brokensep.bbclass
+++ b/classes/autotools-brokensep.bbclass
@@ -1,0 +1,5 @@
+# Autotools class for recipes where separate build dir doesn't work
+# Ideally we should fix software so it does work. Standard autotools supports
+# this.
+inherit autotools
+B = "${S}"

--- a/recipes-devtools/log4cxx/log4cxx_0.10.0.bb
+++ b/recipes-devtools/log4cxx/log4cxx_0.10.0.bb
@@ -15,6 +15,6 @@ SRC_URI[sha256sum] = "0de0396220a9566a580166e66b39674cb40efd2176f52ad2c65486c99c
 
 S = "${WORKDIR}/apache-${BP}"
 
-inherit autotools pkgconfig
+inherit autotools-brokensep pkgconfig
 
 BBCLASSEXTEND += "native"

